test/src/test/java/hudson/model/UpdateSiteTest.java

```diff
@@ -115,19 +118,21 @@ public void setUpWebServer() throws Exception {
         server = new Server();
         ServerConnector connector = new ServerConnector(server);
         server.addConnector(connector);
-        server.setHandler(new AbstractHandler() {
+        server.setHandler(new Handler.Abstract() {
```

Comment:
Adapting to new Jetty 12 APIs as per [the migration guide](https://jetty.org/docs/jetty/12/programming-guide/migration/11-to-12.html).